charts/emby/1.6.3/Chart.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v2
+kubeVersion: ">=1.16.0-0"
+name: emby
+version: 1.6.3
+appVersion:  4.5.4.0
+description: Emby Server is a home media server
+type: application
+deprecated: false
+home: https://github.com/djs52/charts/tree/master/charts/emby
+icon_url: https://raw.githubusercontent.com/djs52/charts/master/charts/emby/icon.png?raw=true
+keywords:
+  - jellyfin
+  - plex
```

Comment:
I don't think it's fair to add keywords for totally different solutions...

Reply:
Heh :) I literally copied this list from the jellyfin chart! I was a bit surprised, but figured it was fair game.

Reply:
Yeah needs to go there too... thanks for the headsup! :)